xtask/src/build_ebpf.rs

```diff
@@ -103,6 +103,7 @@ fn build_c_ebpf(opts: &BuildEbpfOptions) -> anyhow::Result<()> {
 
     let include_path = out_path.join("include");
     get_libbpf_headers(&opts.libbpf_dir, &include_path)?;
+    std::env::set_var("LIBBPF_INCLUDE", &include_path);
```

Comment:
This is  a hack to get libbpf headers inside the RelocationTest builder. I'll gather feedback on the general approach before thinking about a more appropriate solution!